doc/source/whatsnew/v2.2.0.rst

```diff
@@ -97,6 +97,24 @@ This future dtype inference logic can be enabled with:
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_220.enhancements.case_when:
+
+Create a pandas Series based on one or more conditions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :func:`case_when` function has been added to create a Series object based on one or more conditions. (:issue:`39154`)
+
+.. ipython:: python
+
+   import pandas as pd
+
+   df = pd.DataFrame(dict(a=[1, 2, 3], b=[4, 5, 6]))
+   default=pd.Series(['default'],index=df.index)
+   default.case_when(
+        caselist = [(df.a == 1, 'first'),          # condition, replacement
+            (df.a.gt(1) & df.b.eq(5), 'second')],  # condition, replacement
+   )
```

Comment:
```suggestion
   df = pd.DataFrame(dict(a=[1, 2, 3], b=[4, 5, 6]))
   default=pd.Series('default', index=df.index)
   default.case_when(
        caselist=[
            (df.a == 1, 'first'),                              # condition, replacement
            (df.a.gt(1) & df.b.eq(5), 'second'),  # condition, replacement
        ], 
   )
```

Reply:
Thanks @phofl . How did U know this was the issue? I'm guessing this is formatting related?

Reply:
No your series construction was incorrect, your index was longer than your values, I just ran this locally